client/js/options.js

```diff
@@ -80,6 +83,11 @@ settings.on("change", "input, select, textarea", function() {
 			// otherwise, users get notifications for everything
 			return h !== "";
 		});
+	} else if (name === "showSeconds") {
+		var format = options.showSeconds ? constants.timeFormats.msgWithSeconds : constants.timeFormats.msgDefault;
+		chat.find(".msg > .time").each(function() {
+			$(this).text(moment($(this).parent().data("time")).format(format));
```

Comment:
There's a fix for that. Move the import into the function on the export (so it only gets callled when the function gets called) and then add the import for the helper here. It's a little less efficient, but there should be no cyclical dependency.